Makefile

```
@@ -717,7 +717,8 @@ runtime_NATIVE_C_SOURCES = \
   $(runtime_NATIVE_ONLY_C_SOURCES:%=runtime/%.c)
 
 ## Header files generated by configure
-runtime_CONFIGURED_HEADERS = $(addprefix runtime/caml/, m.h s.h version.h)
+runtime_CONFIGURED_HEADERS = \
+  $(addprefix runtime/caml/, m.h s.h version.h exec.h)
```

Consider keeping the files list sorted.
```suggestion
  $(addprefix runtime/caml/, exec.h m.h s.h version.h)
```